src/api/api.tsx

```diff
@@ -0,0 +1,29 @@
+import { CoinsList } from '../types/types';
```

Comment:
This file doesn't contain any React components so there is no point in it having `.tsx` extension, please make it `.ts`

Good job using `const`s where appropriate, though!